{"name": "test_login_params[data_for_login3]", "status": "passed", "attachments": [{"name": "log", "source": "8cd7ec6e-cbe8-4a95-b9ba-ec3c27bbad42-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "data_for_login", "value": "('adelukesh24@gmail.com', '54851', 'Fail')"}], "start": 1745693119088, "stop": 1745693125687, "uuid": "96077cd2-dc6b-47c6-bdc4-384a020bf109", "historyId": "56bb7270e2a590e62dc78e8dce5afe91", "testCaseId": "5fe6fe3249a01c883d8efece63f2c386", "fullName": "test_login_params.TestLoginParams#test_login_params", "labels": [{"name": "severity", "value": "critical"}, {"name": "tag", "value": "regression"}, {"name": "tag", "value": "sanity"}, {"name": "tag", "value": "@pytest.mark.usefixtures('setup', 'log_on_failure')"}, {"name": "suite", "value": "test_login_params"}, {"name": "subSuite", "value": "TestLoginParams"}, {"name": "host", "value": "lukesh"}, {"name": "thread", "value": "16804-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_login_params"}]}